content/en/docs/tutorials/stateless-application/guestbook.md

```diff
@@ -48,341 +51,372 @@ The manifest file, included below, specifies a Deployment controller that runs a
+   ```
 
 1. Query the list of Services to verify that the Redis Service is running:
 
-      ```shell
-      kubectl get service
-      ```
+   ```shell
+   kubectl get service
+   ```
 
-      The response should be similar to this:
+   The response should be similar to this:
 
-      ```shell
-      NAME           TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
-      kubernetes     ClusterIP   10.0.0.1     <none>        443/TCP    1m
-      redis-leader   ClusterIP   10.103.78.24 <none>        6379/TCP   16s
-      ```
+   ```
+   NAME           TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
+   kubernetes     ClusterIP   10.0.0.1     <none>        443/TCP    1m
+   redis-leader   ClusterIP   10.103.78.24 <none>        6379/TCP   16s
+   ```
 
 {{< note >}}
-This manifest file creates a Service named `redis-leader` with a set of labels that match the labels previously defined, so the Service routes network traffic to the Redis Pod.
+This manifest file creates a Service named `redis-leader` with a set of labels
+that match the labels previously defined, so the Service routes network
+traffic to the Redis Pod.
 {{< /note >}}
 
 ### Set up Redis followers
 
-Although the Redis leader is a single Pod, you can make it highly available and meet traffic demands by adding a few Redis followers, or replicas.
+Although the Redis leader is a single Pod, you can make it highly available
+and meet traffic demands by adding a few Redis followers, or replicas.
 
 {{< codenew file="application/guestbook/redis-follower-deployment.yaml" >}}
 
-1. Apply the Redis Service from the following `redis-follower-deployment.yaml` file:
+1. Apply the Redis Deployment from the following `redis-follower-deployment.yaml` file:
 
-      <!---
-      for local testing of the content via relative file path
-      kubectl apply -f ./content/en/examples/application/guestbook/redis-follower-deployment.yaml
-      -->
+   <!---
+   for local testing of the content via relative file path
+   kubectl apply -f ./content/en/examples/application/guestbook/redis-follower-deployment.yaml
+   -->
 
-      ```shell
-      kubectl apply -f https://k8s.io/examples/application/guestbook/redis-follower-deployment.yaml
-      ```
+   ```shell
+   kubectl apply -f https://k8s.io/examples/application/guestbook/redis-follower-deployment.yaml
+   ```
 
 1. Verify that the two Redis follower replicas are running by querying the list of Pods:
 
-      ```shell
-      kubectl get pods
-      ```
+   ```shell
+   kubectl get pods
+   ```
 
-      The response should be similar to this:
+   The response should be similar to this:
 
-      ```shell
-      NAME                             READY   STATUS    RESTARTS   AGE
-      redis-follower-dddfbdcc9-82sfr   1/1     Running   0          37s
-      redis-follower-dddfbdcc9-qrt5k   1/1     Running   0          38s
-      redis-leader-fb76b4755-xjr2n     1/1     Running   0          11m
+   ```
```

Comment:
```suggestion
   ```shell
```

Reply:
@jimangel 
I think @tengqm 's suggestion is more appropriate,
because this block is an output of `kubectl get pods`,
not a block of shell commands or shell script.

PTAL!

Reply:
Excellent point! I retract my suggestion 😁